extensions/authentication-passport/README.md

```diff
@@ -51,11 +58,53 @@ const basicStrategy = new BasicStrategy(verify);
 It's a similar configuration as you add a strategy to a `passport` by calling
 `passport.use()`.
 
-2. Apply the adapter to the strategy
+2. Supply a _user profile factory_ which converts a user to a user profile. It
```

Comment:
maybe change it to `UserProfileFactory` like other places too?

Reply:
The user must supply their own user profile factory that implements that interface. So the way it's worded is ok. :)